gitlab/resource_gitlab_service_jira.go

```diff
@@ -46,6 +46,12 @@ func resourceGitlabServiceJira() *schema.Resource {
 				Required:     true,
 				ValidateFunc: validateURLFunc,
 			},
+			"api_url": {
+				Type:         schema.TypeString,
+				Optional:     true,
```

Comment:
Should it also be computed? Wouldn't the underlying jira give it back anyway?

Do we have a test without the parameter?